ethcore/src/engines/validator_set/simple_list.rs

```diff
@@ -33,9 +33,14 @@ pub struct SimpleList {
 impl SimpleList {
```

Comment:
is simple list used for hardcoded list in specfile as well? (i.e. the case where we don't have a contract, but rather a fixed list from spec)

Reply:
Yes, I believe so: https://github.com/paritytech/parity-ethereum/blob/7c335e87642b848f61ab3ee7bbff5964b53d6f11/ethcore/src/engines/validator_set/mod.rs#L49